src/main/java/bio/terra/cli/command/spend/CreateProfile.java

```diff
@@ -2,17 +2,23 @@
 
 import bio.terra.cli.command.shared.BaseCommand;
 import bio.terra.cli.service.SpendProfileManagerService;
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 /** This class corresponds to the third-level "terra spend create-profile" command. */
-@Command(
-    name = "create-profile",
-    description = "Create the Workspace Manager default spend profile.")
+@Command(name = "create-profile", description = "Create a spend profile.")
 public class CreateProfile extends BaseCommand {
-  /** Create the WSM default spend profile. */
+
+  @CommandLine.Option(
```

Comment:
When an option is needed in more than one command, we usually create a shared option. Let's

- Move this option to shared/options/SpendProfile.java
- This this option out of shared/options/SpendfProfileUser.java
- Use shared/options/SpendProfile.java everywhere